install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/tasks/build_securedrop_app_code_deb.yml

```diff
@@ -33,10 +33,11 @@
 - include: sass.yml
 
 - name: Create pip wheel archive for Debian package requirements.
-  command: >
-    pip wheel
-    -r {{ securedrop_code_filtered }}/requirements/securedrop-app-code-requirements.txt
-    -w {{ securedrop_app_code_deb_dir }}/var/securedrop/wheelhouse
+  shell: |
+    pip wheel \
+      -r {{ securedrop_code_filtered }}/requirements/securedrop-app-code-requirements.txt \
+      -w {{ securedrop_app_code_deb_dir }}/var/securedrop/wheelhouse 2>&1 | tee /tmp/w.out
+      ! grep -i --quiet 'Failed to build' /tmp/w.out
```

Comment:
Some super fancy ansible pros would gasp at the usage of error handling like this.... luckily, I think this is fine to do this here ;) Ive seen others break this logic up over two tasks (one task runs, the other task checks the output of the first and fails conditionally with text present). I honestly think the way you went makes more sense.

Reply:
:-) I'm glad we agree on this.